### PR TITLE
Fix the README description

### LINKED ISSUE #73: update readme to remove reference to Ember.select
Really minor, but Ember.Select is deprecated, so a new justification for this component in the readme would be helpful. The new suggested way to do selects in the ember doc is also not great....

### PR DESCRIPTION
**Pull request by Robdel12**
This should close #73

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,10 @@
...
 
 
-Why a select component? Because `Ember.Select` is hard to use, and
-really, all you want to do is just use a `<select>` tag dammit. The
-only difference is that you want value of the options to be any object
-and not just a string.
+Why a select component? Because we've done the hard work and built a
+select component for you. Sometimes all you want to do is just use a
+`<select>` tag dammit. The only difference is that you want value of
+the options to be any object and not just a string.
```
Comment (Robdel12) [inline README.md]: Calling upon @tehviking for your words. Your words are the best. I could use your words here. I do not have the best words.

Comment (tehviking) [inline README.md]: There are a lot of options out there for fancy dropdowns, but after spending many months writing them, we've learned you can't beat a native `<select>` for reliability, accessibility, and low cost of maintenance. `<x-select>` is a drop-in component to let you use any object for your selectable options.

Comment (tehviking) [inline README.md]: @Robdel12 something like that. Feel free to edit.

File: README.md [hunk 2]
```diff
@@ -6,11 +6,17 @@
 
 A Select component based on the native html select.
 
+There are a lot of options out there for fancy dropdowns, but after
+spending many months writing them, we've learned you can't beat a
+native `<select>` for reliability, accessibility, and low cost of
+maintenance. `<x-select>` is a drop-in component to let you use any
+object for your selectable options. You can use it out of the box, or
+as a building block of something more ambitious.
+
```
Comment (flexyford) [inline README.md]:
```
There are a lot of options out there for fancy dropdowns, but after
 +spending many months writing them, we've learned you can't beat a
 +native `<select>` for reliability, accessibility, and low cost of
 +maintenance.
```

==>

```
`ember-xselect` provides the reliability, accessibility of a native `<select>`.
```

**Pull request merged**